tests/publiccloud/instance_overview.pm

```diff
@@ -55,5 +60,9 @@ sub run {
     zypper_call("lr");
 }
 
+sub test_flags {
+    return {fatal => 1};
```

Comment:
any good reason why you want to cancel everything in case of some issues here ?

Reply:
See the [progress issue](https://progress.opensuse.org/issues/80714) - `instance_overview` acts as a health check for the image, if anything is wrong here it has little sense to go on because then there is a major issue.

Reply:
ok